source/localizable/feedspec/index.en.html.md.erb

```diff
@@ -661,8 +884,75 @@ The XML excerpt below is an example of a compatible XML data feed.
 </mywebstore>
 ~~~
 
+### XML with Size Variations
 
-## XSD
+The XML excerpt below is an example of a compatible XML with variations data feed.
+
+~~~ xml
+<?xml version="1.0" encoding="UTF-8"?>
+<mywebstore>
+   <created_at>2010-04-08 12:32</created_at>
+   <products>
+      <product>
+        <id>322233</id>
+        <name><![CDATA[Converse All Star Chuck Taylor Ox]]></name>
+        <link><![CDATA[https://www.mywebstore.co.uk/product/322233]]></link>
+        <image><![CDATA[https://www.mywebstore.co.uk/product/322233.jpg]]></image>
+        <additionalimage><![CDATA[https://www.mywebstore.co.uk/product/322233/front.jpg]]></additionalimage>
+        <category><![CDATA[Outdor > Sneakers]]></category>
+        <price_with_vat>30.00</price_with_vat>
+        <vat>24.00</vat>
+        <manufacturer><![CDATA[Converse]]></manufacturer>
+        <mpn>M7652C</mpn>
+        <ean>886952780692</ean>
+        <instock>N</instock>
+        <availability>Delivery 1 to 3 days</availability>
+        <size>43,45</size>
+        <weight>2000</weight>
+        <color>White</color>
+        <description>
+          The Chuck Taylor All Star is one of the most iconic sneakers
+          of all ages. It is a unique design with a classic and comfortable
+          fit, in white color, with durable non-slip exterior rubber sole.
+          It is an ideal piece for all kinds of appearances.
+        </description>
+        <quantity>10</quantity>
+        <variations>
+          <variation>
+            <variationid>322233.43</variationid>
+            <link><![CDATA[https://www.mywebstore.co.uk/product/322233]]></link>
+            <availability>Delivery 1 to 3 days</availability>
+            <manufacturersku>M7652C</manufacturersku>
+            <ean>886952780692</ean>
+            <price_with_vat>33.00</price_with_vat>
+            <size>43</size>
+            <quantity>6</quantity>
+          </variation>
+          <variation>
+            <variationid>322233.45</variationid>
+            <link><![CDATA[https://www.mywebstore.co.uk/product/322233]]></link>
+            <availability>Delivery 1 to 3 days</availability>
+            <manufacturersku>M7652C</manufacturersku>
+            <ean>886952780692</ean>
+            <price_with_vat>30.00</price_with_vat>
+            <size>45</size>
+            <quantity>4</quantity>
+          </variation>
+        </variations>
+        ...
+        ...
+        ...
```

Comment:
[minor] I think these dots must be placed inside of the `<variations>` element like `<products>`.